scripts/maint/practracker/metrics.py

```diff
@@ -16,21 +20,28 @@ def get_include_count(f):
             include_count += 1
     return include_count
 
-def function_lines(f):
+def get_function_lines(f):
     """
     Return iterator which iterates over functions and returns (function name, function lines)
     """
 
-    # XXX Buggy! Doesn't work with MOCK_IMPL and ENABLE_GCC_WARNINGS
+    # Skip lines with these terms since they confuse our regexp
+    REGEXP_CONFUSE_TERMS = ["MOCK_IMPL", "ENABLE_GCC_WARNINGS", "ENABLE_GCC_WARNING", "DUMMY_TYPECHECK_INSTANCE",
```

Comment:
I'd suggest that you use a set here rather than an array.

Reply:
Fixed in ac05cc7437.